### PR TITLE
Add bios for @ryanuber, @slackpad, and @sethvargo

### PR DESCRIPTION
**Pull request by sethvargo**
/cc @armon

### CODE REVIEW

File: website/source/community.html.erb [hunk 1]
```diff
@@ -94,6 +94,59 @@ description: |-
...
+
+				<div class="clearfix"></div>
+
+				<div class="person">
+					<img class="pull-left" src="//avatars1.githubusercontent.com/u/408570?v=3&amp;s=125">
+					<div class="bio">
+						<h3>Seth Vargo (<a href="https://github.com/sethvargo">@sethvargo</a>)</h3>
+						<p>
+						Seth Vargo is a contributor to Consul, but his main focus is the
+						<a href="https://www.consul.io/downloads_tools.html">Consul tools ecosystem</a>
```
Comment (ryanuber) [inline website/source/community.html.erb]: Need a period.

**Comment by ryanuber** [comment]
Woah, didn't realize we never updated this! Thanks @sethvargo, LGTM.

File: website/source/community.html.erb [hunk 2]
```diff
@@ -94,6 +94,59 @@ description: |-
...
+
+				<div class="clearfix"></div>
+
+				<div class="person">
+					<img class="pull-left" src="//avatars3.githubusercontent.com/u/673509?v=3&amp;s=125">
+					<div class="bio">
+						<h3>James Phillips (<a href="https://github.com/slackpad">@slackpad</a>)</h3>
+						<p>
+						James Phillips is a core contributor to Consul where he works on all
+						facets of Consul, including network tomography and preparred
```
Comment (tgwizard) [inline website/source/community.html.erb]: Typo: s/preparred/prepared/ :smile:

**Comment by slackpad** [comment]
Thanks @sethvargo (and @tgwizard I fixed that)!

**Pull request merged**